backend/src/test/java/heartbeat/service/report/KanbanCsvServiceTest.java

```diff
@@ -618,7 +619,79 @@ void shouldAddFixedFieldsWithCorrectValueFormatWhenCustomFieldValueInstanceOfLis
 	}
 
 	@Test
-	void shouldAddReworkFieldsWhenGenerateSheetGivenReworkStateAndExcludedStates() throws URISyntaxException {
+	void shouldAddReworkFieldsWhenGenerateSheetGivenReworkStateAndExcludedStatesAndConsiderFlagAsBlock()
+			throws URISyntaxException {
+		URI uri = new URI("site-uri");
+		when(urlGenerator.getUri(any())).thenReturn(uri);
+		when(jiraService.getJiraBoardConfig(any(), any(), any())).thenReturn(JiraBoardConfigDTO.builder().build());
+		when(jiraService.getJiraColumns(any(), any(), any())).thenReturn(JiraColumnResult.builder()
+			.jiraColumnResponse(List
+				.of(JiraColumnDTO.builder().value(ColumnValue.builder().statuses(List.of("BLOCKED")).build()).build()))
+			.build());
+		JiraCard jiraCard = JiraCard.builder().fields(MOCK_JIRA_CARD()).build();
+		JiraCard jiraCard2 = JiraCard.builder().fields(MOCK_JIRA_CARD()).build();
+		jiraCard2.getFields().setLastStatusChangeDate(1701251323000L);
```

Comment:
Change `1701251323000L` as a readable variable, like 2024-01-01 12:30:30

Reply:
can not change 1701251323000L, because LastStatusChangedDate is from frontend, and it's type is Long